plugins/inputs/vsphere/README.MD

```diff
@@ -0,0 +1,350 @@
+  ## when true for large environments this may cause errors for time elapsed while collecting metrics
+  ## when false (default) the first collection cycle may result in no or limited metrics while objects are discovered
+  # force_discover_on_init = false
+
+  ## the interval before (re)discovering objects subject to metrics collection (default: 300s)
+  # object_discovery_interval = "300s"
+
+  ## timeout applies to any of the api request made to vcenter
+  # timeout = "20s"
+
+  ## Optional SSL Config
+  # ssl_ca = "/path/to/cafile"
+  # ssl_cert = "/path/to/certfile"
+  # ssl_key = "/path/to/keyfile"
+  ## Use SSL but skip chain & host verification
+  # insecure_skip_verify = false
+``` 
+
+### Objects and Metrics Per Query
+
+Default settings for vCenter 6.5 and above is 256. Prior versions of vCenter have this set to 64. A vCenter administrator
+can change this setting, which should be reflected in this plugin. See this [VMware KB article](https://kb.vmware.com/s/article/2107096)
+for more information.
+
+### Collection and Discovery concurrency
+
+On large vCenter setups it may be prudent to have multiple concurrent go routines collect performance metrics
+in order to avoid potential errors for time elapsed during a collection cycle. This should never be greater than 8,
+though the default of 1 (no concurrency) should be sufficient for most configurations. 
+
+## Measurements &amp; Fields
+
+- Cluster Stats
+	- Cluster services: CPU, memory, failover
+	- CPU: total, usage
+	- Memory: consumed, total, vmmemctl
+	- VM operations: # changes, clone, create, deploy, destroy, power, reboot, reconfigure, register, reset, shutdown, standby, vmotion
+- Host Stats:
+	- CPU: total, usage, cost, mhz
+	- Datastore: iops, latency, read/write bytes, # reads/writes
+	- Disk: commands, latency, kernel reads/writes, # reads/writes, queues
+	- Memory: total, usage, active, latency, swap, shared, vmmemctl
+	- Network: broadcast, bytes, dropped, errors, multicast, packets, usage
+	- Power: energy, usage, capacity
+	- Res CPU: active, max, running
+	- Storage Adapter: commands, latency, # reads/writes
+	- Storage Path: commands, latency, # reads/writes
+	- System Resources: cpu active, cpu max, cpu running, cpu usage, mem allocated, mem consumed, mem shared, swap
+	- System: uptime
+	- Flash Module: active VMDKs 
+- VM Stats:
+	- CPU: demand, usage, readiness, cost, mhz
+	- Datastore: latency, # reads/writes
+	- Disk: commands, latency, # reads/writes, provisioned, usage
+	- Memory: granted, usage, active, swap, vmmemctl
+	- Network: broadcast, bytes, dropped, multicast, packets, usage
+	- Power: energy, usage
+	- Res CPU: active, max, running
+	- System: operating system uptime, uptime
+	- Virtual Disk: seeks, # reads/writes, latency, load 
+- Datastore stats:
+	- Disk: Capacity, provisioned, used  
+
+For a detailed list of commonly available metrics, please refer to [METRICS.MD](METRICS.MD)
+	
+## Tags
```

Comment:
I will probably merge the Tags section with Measurements & Fields when I merge the PR, unless you do it first :)

Check [EXAMPLE_README.md](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md) for the latest style.